src/components/side-panel/layer-panel/layer-configurator.js

```diff
@@ -843,6 +842,60 @@ export default function LayerConfiguratorFactory(SourceDataSelector) {
       );
     }
 
+    _renderS2LayerConfig({
```

Comment:
if s2 and h3 has exact same configuration, we should create a `_renderColumnConfig` method and call from both. same as how  `arc` and `line` layer share configuration render method

Reply:
s2 doesn't have the coverage configuration property based on this: https://github.com/uber/deck.gl/blob/master/modules/geo-layers/src/s2-layer/s2-layer.js